---
title: "Analyzing the effects of DMARD treatment on genetic expression"
format:
  revealjs:
    embed-resources: true
    theme: "sky"
    transition: slide
    background-transition: fade
editor: visual
author: "Anna Boateng (s175562), Marlies Goedknegt (s220675), Anne Gjelstrup (s194530), Katja Jagd (s185395), Olivia Ronnenberg (s183359)"
---

# Introduction

## Background Information

The study looked into genome-wide transcriptional effects in RA (rheumatoid arthritis) patients who are taking several disease-modifying drugs (DMARDs).

For this purpose, synovial tissue biopsy samples were taken from nineteen RA patients, before and after six months of DMARD treatment.

It is hypothesized that multiple biological pathways are impacted by DMARD, but specifics of the pathways are unknown.

## Study's findings

From the 8000 analyzed genes, less than 300 genes had a reversed gene expression as a result of DMARD therapy. These genes are related to the immune system, and largely expressed by lymphocytes.

## Project purpose

Can we identify a molecular pathway that is influenced by DMARD treatment? And more specifically, can we identify relevant treatment targets, depending on their respective molecular pathway(s).

# Materials and Methods

## Preparing the data

From the available data, we are interested in Supplementary tables 1 and 3, containing co-expression modules, and gene expression regulator molecules, respectively.

The data in supplementary table 3 consisted of two separate parts, split into the genetic patient data before and after treatment.

## Wrangling the data

In order to perform our analysis, some data wrangling is required. In this process, we split, pivoted, joined, the data of interest. We furthermore removed redundant data.

## Adding further required variables

For our initial data frame, we want to distinguish between the regulator molecules, which are significantly expressed and which are not.

For our analysis of significant regulator molecules, we want to distinguish between genetic up- and down-regulation. We did this by adding a factor variable, dividing the genes into the two distinctive groups.

# Results

## Differential expressed genes

To visualize the genetic up- and down-regulation, two volcano plots were made. These plots indicate the differences in genetic expression before and after DMARD treatment, respectively.

## Differential expressed genes before treatment

First, the plot below shows the genetic expression levels in RA patients, before DMARD treatment.

![](images/05_expr_plot_before-02.png){fig-align="center"}

## Differential expressed genes after treatment

Secondly, we show genetic expression levels in RA patients, after DMARD treatment.

![](images/05_expr_plot_after-01.png){fig-align="center"}

## Differential expressed genes after treatment

Lastly, we show a comparison plot to indicate genetic differences as a result of treatment.

![](images/05_comparison_plot.png){fig-align="center"}

## Target molecules

The upstream regulators each influence several target molecules. To analyze which regulators had the most severe impact on molecular pathways, we plot the number of target molecules, affected by each regulator.

## Target molecules

![](images/06_targets_plot.png){fig-align="center"}

## Target molecules

The regulators S100A9, TGFB1, and TLR2, are found to undergo significant expression changes due to DMARD treatment.

Whether or not these genes have a sufficient impact on RA prognosis, depends on the molecular pathway they play a role in. Therefore, we analyzed which molecular mechanisms the gene is involved in.

## Molecular mechanisms related to the most influential genes

![](images/06_modules_plot.png){fig-align="center"}

## Molecular mechanisms related to the most influential genes

From this plot, TGFB1 can be a potential drug target.

# Discussion

## Findings from the study

The study by Walsh et.al. found 292 genes down regulated due to treatment, and 23 up regulated. The down regulated genes are related to the immune system, a.o. T-cell activating-genes.

This can be explained due to the fact that RA is an auto-immune disease. Less disease activity means less immune activity, and vice versa.

## Our findings

We identified TGFB1 as a potential drug target.
